### DISCUSSION TITLE
Github badges

### DISCUSSION THREAD

**Discussion by Jakhaishya** [community/community#120439] [New to GitHub]
how to get github badges

**Comment by PrakharChauhan1739** [comment]
dont know

**Comment by ladunjexa** [comment]
https://github.com/drknzz/GitHub-Achievements

**Comment by luanmarcal** [comment]
See this repository, it explains in detail how to achieve it:

[Repository-Badges](https://github.com/Thinkright20/Profile-Badges)

The owner of the repository also made a video showing some achievements:

[Tutorial-Video](https://youtu.be/oJWFfp3sGYA?si=0fyr2lHuH7g2yqw4)

If the answer helps you please mark this answer as the correct answer.

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
